src/MoviePage.js

```diff
@@ -0,0 +1,40 @@
+import React, { Component } from "react";
+// import './MoviePage.scss';
+import { getSingleMovieDetails } from "./APIRequests";
+
+class MoviePage extends Component {
+  constructor() {
+    super();
+    this.state = {};
+  }
+
+  // componentDidMount() {
```

Comment:
Nice work going after these methods! Looking forward to getting `componentDidMount()` and `getMovieDetails` working today!